src/components/scenes/RequestScene.js

```diff
@@ -334,13 +341,13 @@ export class RequestComponent extends React.PureComponent<Props, State> {
               </InputAccessoryView>
             ) : null}
 
-            <View style={styles.qrContainer}>
-              <QrCode data={this.state.encodedURI} size={theme.rem(10)} />
+            <View style={[styles.qrContainer, { width: this.state.qrCodeContainerHeight }]} onLayout={this.handleQrCodeLayout}>
```

Comment:
check to use

overflow: "auto",
width: "100%"

Reply:
Does not work, width '100%' makes it stretch all the way, theres overflow on rn does not have auto

Reply:
Ok, was worth a shot :)